.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2.1
+
+orbs:
+  dwave: dwave/orb-examples@2
+
+workflows:
+  version: 2.1
+  tests:
+    jobs:
+      - dwave/test-linux
+      - dwave/test-osx
+      - dwave/test-win
+
+  daily:
+    triggers:
+      - schedule:
+          cron: "0 18 * * *"
```

Comment:
How about we start with weekly runs here?

Reply:
Sounds good! I wasn’t entirely sure if we had fully decided on weekly runs of all python versions. Since I don’t think there are any objections, I’ll go ahead and do that.